packages/teams-js/test/public/storage.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ErrorCode } from '../../src/public';
+import { app } from '../../src/public/app';
+import { storage } from '../../src/public/storage';
+import { Utils } from '../utils';
+import {FrameContexts,HostClientType } from '../../src/public/constants';
+import { FramelessPostMocks } from '../framelessPostMocks';
+
+describe('storage', () => {
+  const mobilePlatformMock = new FramelessPostMocks();
```

Comment:
See this is never used to initialize with context only used in `beforeEach` and `afterEach` section. There should be two sets of tests i.e., `Framed (Utils) and Frameless (FramelessPostMocks)` can you please add the remaining ones?

Reply:
added UT for both Framed and Frameless.